package.json

```diff
@@ -15,6 +15,12 @@
     "postinstall": "make all",
     "start": "node main.js"
   },
+  "husky": {
+    "hooks": {
```

Comment:
Please also add a `pre-commit` hook that prevents commiting to the master branch directly to enforce best practices and prevent GitHub rejecting your push. Something like this:

```bash
#!/bin/sh

protected_branch='master'
current_branch=$(git symbolic-ref HEAD | sed -e 's,.*/\(.*\),\1,')

if [ $protected_branch = $current_branch ]
then
    echo "$current_branch is protected, so commiting to it is not allowed."
    exit 1 # commit will not execute
else
    exit 0 # commit will execute
fi
```